### PR TITLE
CB-239: Reference review style guide when writing a review

### PR DESCRIPTION
**Pull request by naiveaiguy**
[Jira ticket](https://tickets.metabrainz.org/browse/CB-239)

Minor change - just added it before the "you can use markdown" message.

### CODE REVIEW

File: critiquebrainz/frontend/templates/review/modify/base.html [hunk 1]
```diff
@@ -23,8 +23,10 @@
     </div>
     <div class="form-group">
       <small class="text-muted pull-left" style="font-style:italic;">
-        {{ _('You can use <a href="%(url)s" target="_blank">Markdown</a> syntax to apply custom formatting.',
-             url='http://daringfireball.net/projects/markdown/syntax') }}
+          {{ _('Make sure to keep the <a href="%(style_guide_url)s" target="_blank">style guide</a> in mind when writing.
```
Comment (ferbncode) [inline critiquebrainz/frontend/templates/review/modify/base.html]: I think we can write the statement such that it doesn't seem as a strict compulsion/constraint. Something like:
"Read the [style guide](https://wiki.musicbrainz.org/How_to_write_Good_Critiquebrainz_Reviews) for writing good CritiqueBrainz reviews!"

Comment (eshansingh) [reply critiquebrainz/frontend/templates/review/modify/base.html]: Amended commit.

File: critiquebrainz/frontend/templates/review/modify/base.html [hunk 2]
```diff
@@ -23,8 +23,10 @@
     </div>
     <div class="form-group">
       <small class="text-muted pull-left" style="font-style:italic;">
-        {{ _('You can use <a href="%(url)s" target="_blank">Markdown</a> syntax to apply custom formatting.',
-             url='http://daringfireball.net/projects/markdown/syntax') }}
+          {{ _('You can read the <a href="%(style_guide_url)s" target="_blank">style guide</a> for writing good CritiqueBrainz reviews!
+                Remember, you can use <a href="%(markdown_url)s" target="_blank">Markdown</a> syntax to apply custom formatting.',
```
Comment (ferbncode) [inline critiquebrainz/frontend/templates/review/modify/base.html]: Also, I suggest we change `Remember` to  `Also` or `Furthermore`.
Also, the next sentence (about using `markdown`) gets little out of focus. We can make use of a `<br/>` bringing it to a new line.

Comment (eshansingh) [reply critiquebrainz/frontend/templates/review/modify/base.html]: Amended.

**Pull request merged**